tests/vmcheck/test-misc-1.sh

```diff
@@ -48,10 +48,15 @@ vm_assert_status_jq \
   '.deployments[0]["requested-local-packages"]' \
   '.deployments[0]["base-removals"]' \
   '.deployments[0]["requested-base-removals"]' \
-  '.deployments[0]["base-commit-meta"]["ostree.source-title"]|contains("overlay")' \
   '.deployments[0]["layered-commit-meta"]|not'
 echo "ok empty pkg arrays, and commit meta correct in status json"
 
+if test -z "${RPMOSTREE_TEST_NO_OVERLAY}"; then
```

Comment:
I think this needs to have `:-` so we don't fail if it's not set.

Reply:
Yeah, see https://s3.amazonaws.com/aos-ci/ghprb/projectatomic/rpm-ostree/0e88e38b37e1722c7facaae53dd5e50cbd41d4eb.2.1536697535446784971/artifacts/vmcheck/misc-1.log

Reply:
My bash variable fu is weak